pdr_backend/trader/approach1/trader_agent1.py

```diff
@@ -0,0 +1,129 @@
+import random
```

Comment:
I like that you're implementing this by inheriting from the base agent, and creating approach1. Perfect.

That's in line with what predictoor does, and what [pdr-backend#231](https://github.com/oceanprotocol/pdr-backend/issues/231) anticipates: "[trader bot] Streamline strategy implementation, introduce backtesting". Doing it here first (for issue 275) is a nice precursor to 231. Great!

Reply:
Thank you for taking this from the depths of branches and deploying a draft, and providing feedback. Cheers!